consent-ws/src/main/java/org/broadinstitute/consent/http/MongoHealthCheck.java

```diff
@@ -1,20 +1,33 @@
 package org.broadinstitute.consent.http;
 
 import com.codahale.metrics.health.HealthCheck;
-import com.mongodb.Mongo;
+import com.mongodb.MongoClient;
+import com.mongodb.BasicDBObject;
+import com.mongodb.MongoException;
 
 public class MongoHealthCheck extends HealthCheck {
  
-    private final Mongo mongo;
+    private final MongoClient mongo;
+    private final String databaseName;
  
-    protected MongoHealthCheck(Mongo mongo) {
+    protected MongoHealthCheck(MongoClient mongo, String databaseName) {
         super();
         this.mongo = mongo;
+        this.databaseName = databaseName;
     }
  
     @Override
     protected Result check() throws Exception {
-        mongo.getDatabaseNames();
```

Comment:
`getDatabaseNames` is deprecated, so I replaced it with a [command](https://docs.mongodb.com/manual/reference/command/ping/) recommended by the MongoDB documentation (also referenced in the SO link)